src/app/zeko/da-layer/contracts/Types.sol

```diff
@@ -1,6 +1,22 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.8.13;
 
+/**
+ * @dev Multisig transaction proposal, for validators to vote on and
+ * execute.
+ */
+struct Transaction {
+    address destination;
+    uint256 value;
+    bytes data;
+    bool executed;
+    uint256 validatorVotePeriod;
+}
```

Comment:
What do these fields represent?

Reply:
It's the multisig. The original idea was to reuse the multisig from Milkomeda and adjust it for our needs. The original source code is here https://github.com/dcSpark/milkomeda-validator/tree/main/m1/contracts/src.

Reply:
Why do you need this though?

Reply:
it's for adding/removing DA validators

Reply:
Please add comments for this then